src/DIRAC/FrameworkSystem/DB/UserProfileDB.py

```diff
@@ -62,17 +58,6 @@ class UserProfileDB(DB):
                                                },
                                    'Engine': 'InnoDB',
                                    },
-               'up_HashTags': {'Fields': {'UserId': 'INTEGER',
```

Comment:
Was this table ever used?

Reply:
Not that I can easily find with `git grep`